static/faq.html

```diff
@@ -291,6 +293,22 @@ <h2 id="default-connections">
                 </li>
             </ul>
 
+            <h2 id="cellular-tracking">
+                <a href="#cellular-tracking">What does GrapheneOS do about cellular tracking and
+                silent SMS?</a>
+            </h2>
+
+            <p>GrapheneOS always considers the network to be hostile. Merely getting a silent SMS is
+            not a good indicator of being targeted by your cell carrier, police or government
+            because <i>anyone on the cell network can send them</i>. Cellular triangulation will
```

Comment:
You should use `<em></em>`, not `<i></i>` which is deprecated in modern HTML. Similarly, use `<strong></strong>` rather than `<b></b>`.

Reply:
Whoops! Okay! Let me go and fix that...

Reply:
https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-em-element vs. https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-i-element

https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-strong-element vs. https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-b-element

Essentially, em and strong are for the semantic meanings associated with those, and i and b are for more subtle cases without those clear semantic meanings but rather a more subtle reason. For a long time, i and b were deprecated, but they came up with new meanings for them giving them purpose again and they aren't deprecated anymore.

Reply:
It matters for accessibility, since accessibility tools can read the text in a different way for em and strong, but the others are more subtle.

Reply:
I think this may appear in a couple other places, I'll go get them too.